test/observe_test_utils.dart

```diff
@@ -7,8 +7,27 @@ library observe.test.observe_test_utils;
 import 'dart:async';
 import 'package:observe/observe.dart';
 import 'package:observe/mirrors_used.dart' as mu; // to make tests smaller
-import 'package:unittest/unittest.dart';
+import 'package:observe/src/dirty_check.dart';
+import 'package:test/test.dart' hide test, group, setUp, tearDown;
+import 'package:test/test.dart' as original_test
+    show test, group, setUp, tearDown;
+
 export 'package:observe/src/dirty_check.dart' show dirtyCheckZone;
+export 'package:test/test.dart' hide test, group, setUp, tearDown;
+
+/// Custom implementations of the functions from `package:test`. These ensure
+/// that the body of all test function are run in the dirty checking zone.
+test(String description, body()) => original_test.test(
+    description, () => dirtyCheckZone().bindCallback(body)());
```

Comment:
I just mean a top level function which contained all the invocations of `test`, see https://github.com/dart-lang/observe/pull/84/files#diff-d0b4c4bde478579a03ae3898a56bb2edL13.